lib/jolie-xml/src/jolie/xml/XmlUtils.java

```diff
@@ -114,6 +121,76 @@ private static void addForcedAttribute( Value value, Element element ) {
 			value.getFirstChild( FORCE_ATTRIBUTE ).children().forEach( action );
 		}
 	}
+
+	
+	public static Transformer valueToDocument( 
```

Comment:
This should really be a `void` method, because all the other `valueToDocument` are void. If you need to pass a transformer around, shouldn't it be a parameter of this method instead?

This indicates that we should discuss how to refactor this library/service.

Reply:
The transformer is used in different ways in FileService and XmlUtils.java (the javaservice). Maybe we could improve it by passing a lambda which executes a different code.